src/webhook/enums/webhookName.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum WebhookName {
+  RECIPE_CREATED_WEBHOOK_NAME = 'RecipeCreatedWebhook',
```

Comment:
Please change the enum namings
Typical naming convention would be 
export enum WebhookName {
RecipeCreatedWebhook = 'recipeCreated'
...
https://michaelcurrin.github.io/dev-cheatsheets/cheatsheets/typescript/data-types/enums.html#:~:text=Naming%20convention,enum%20tend%20to%20be%20titlecase.